code/game/machinery/hologram.dm

```diff
@@ -41,313 +37,275 @@ var/const/HOLOPAD_MODE = RANGE_BASED
 	idle_power_usage = 5
 	use_power = 1
 
```

Comment:
Didn't you lose the original description in the process? (see deleted line [64](https://github.com/Aurorastation/Aurora.3/pull/10370/files#diff-699300d195cf7e4480207b2c408491d4ae5047e1ab3416a7d7309e58d467dc01L64) / new [57](https://github.com/Aurorastation/Aurora.3/pull/10370/files#diff-699300d195cf7e4480207b2c408491d4ae5047e1ab3416a7d7309e58d467dc01R57))

I'd set the holopad's initial desc to something like
```suggestion
	desc = "A floor-mounted device for projecting holographic images."

```

Reply:
line 31 still has `desc = "It's a floor-mounted device for projecting holographic images."`, the one in Initialize just adds that bit onto the desc.

Reply:
Oh I see, it was just exactly hidden by the github diff. Sorry.

Reply:
Oh I see, it was just exactly hidden by the github diff. Sorry.